Clinical-Genomics/preClinVar
# Updated setuptools to v.71 to fix a security issue

## Description
### Fixed
- Closes #128

### How to test
- Deploy and start locally
- Make sure the image is pushed to Docker Hub

## Review

Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
